src/snowflake/snowpark/_internal/compiler/utils.py

```diff
@@ -382,6 +382,19 @@ def get_name(node: Optional[LogicalPlan]) -> str:
             name = f"{name} :: ({get_name(node.snowflake_plan.source_plan)})"
         elif isinstance(node, SetStatement):
             name = f"{name} :: ({node.set_operands[1].operator})"
+        elif isinstance(node, SelectStatement):
+            properties = []
```

Comment:
adding some more details to `SelectStatement` by highlighting attributes that are set for this node.

Reply:
Why do we want so many details here? What would be the benefit?

Reply:
I wanted to see if select statement was breaking with only projection or if it was because of sort. so added all details here.